.travis.yml

```diff
@@ -294,7 +294,7 @@ os: linux
       - bash -c "${TRAVIS_BUILD_DIR}/contrib/ci-horizen/scripts/common/travis_stay_alive.sh"
       - bash -c '${TRAVIS_BUILD_DIR}/contrib/ci-horizen/scripts/test/run_test.sh "${TEST_CMD}" "${TEST_ARGS}"'
   - &test_osx_xcode9_4
-    if: (env(SKIP_TESTS) != true) AND (env(SKIP_OSX) != true)
+    if: (NOT env(SKIP_TESTS)) AND (NOT env(SKIP_OSX)) OR tag IS present # Never skip MacOS when running the CI for a tag
```

Comment:
As for C/C++, I believe `OR` operator has lower precedence than `AND` operator, but if that's not the case we should modify with:

`if: ((NOT env(SKIP_TESTS)) AND (NOT env(SKIP_OSX))) OR (tag IS present) # Never skip MacOS when running the CI for a tag`

otherwise just skip this comment.

Reply:
Looking at the [documentation](https://docs.travis-ci.com/user/conditions-v1#boolean-operators) it seems like Travis has standard precedence rules, so I've just completely removed the redundant square brackets.